libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,130 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface HubspotFormField {
+    name: string;
+    value: string | any;
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    selectedValue?: string;
+    userEmail: string;
+    clientId: string | number;
+    dialogTitle: string;
+    cancelButtonText?: string;
+    showCancelButton?: boolean;
+    onClose: () => void;
+    onFormSubmitted: () => void;
+}
+
+/**
+ * @internal
+ */
+export const ConversionTouchPointDialogBase: React.FC<IConversionTouchPointDialogBaseProps> = (props) => {
+    const HUBSPOT_EMAIL_FIELD_NAME = "email";
+    const HUBSPOT_GACID_FIELD_NAME = "gacid";
+    const HUBSPOT_PORTAL_ID = "8934247";
+    const HUBSPOT_FORM_ID = "94ade70f-4052-432d-9453-5cbf26981926";
```

Comment:
Values of `HUBSPOT_PORTAL_ID` and `HUBSPOT_FORM_ID` should not be put in SDK, it should be received from Our App.(AD, KD, ....).
And we will note for Users to know to have to register account of Hubspot to using.
https://github.com/gooddata/gooddata-ui-sdk/blob/1fe2088a8be6f7c0d15158828b9eec82aa2f07c0/libs/sdk-ui-ext/src/internal/dashboardEmbedding/types.ts#L66

Reply:
Updated. Thanks!